examples/src/main/scala/caliban/play/ExampleApp.scala

```diff
@@ -0,0 +1,45 @@
+package caliban.play
+
+import play.api.Mode
+import play.api.mvc.{DefaultControllerComponents, Handler, RequestHeader}
+import play.api.routing.sird._
+import play.core.server.AkkaHttpServer
+import play.core.server.ServerConfig
+
+import scala.io.StdIn.readLine
+
+object ExampleApp extends App {
+
+  val server = AkkaHttpServer.fromRouterWithComponents(
+    ServerConfig(
+      mode = Mode.Dev, 
+      port = Some(8088),
+      address = "127.0.0.1"
+    )
+  ) { components =>
+    val controller = new CalibanController(
+      DefaultControllerComponents(
+        components.defaultActionBuilder,
+        components.playBodyParsers,
+        components.messagesApi,
+        components.langs,
+        components.fileMimeTypes,
+        components.executionContext
+      )
+    )(components.actorSystem, components.materializer)
+    val router: PartialFunction[RequestHeader, Handler] = {
+      case POST(p"/api/graphql") => controller.graphqlPost
+      case GET(
+          p"/api/graphql" ? q"query=$query" & q_o"variables=$variables" & q_o"operation=$operation" & q_o"extensions=$extensions"
+          ) =>
+        controller.graphqlGet(query, variables, operation, extensions)
+      case GET(p"/ws/graphql") => controller.webSocket()
```

Comment:
Couldn't we just roll this up into a `Router` ourselves?

Reply:
Do you mean using [Sird Router](https://www.playframework.com/documentation/2.8.x/ScalaSirdRouter#Binding-sird-Router)?

Reply:
Yeah, I'm thinking the easiest out-of-the-box setup would just be like:

```
val router = CalibanRouter(
  subscriptions = true,
  playground = true,
  allowGETRequests = true
)
```

Reply:
Keep the others accessible, because people might want to manipulate configuration, but that would allow pretty much drop in usage.

Reply:
ahh got it, it's a good point. I'll try to do it.

Reply:
I've added a PlayRouter. Could you check my last commit out? and let me know what do you think.